Description: Do not unconditionally add -fPIC, this is only done for the shared libs.
Forwarded: not-needed
Author: Peter Pentchev <roam@FreeBSD.org>
Last-Update: 2009-11-26

--- a/cdb/Makefile
+++ b/cdb/Makefile
@@ -43,7 +43,7 @@
 compile: \
 warn-auto.sh conf-cc
 	( cat warn-auto.sh; \
-	echo exec "`head -1 conf-cc`" -fPIC '-c $${1+"$$@"}' \
+	echo exec "`head -1 conf-cc`" '-c $${1+"$$@"}' \
 	) > compile
 	chmod 755 compile
 
